### PR TITLE
pattern(global header): incorporates feedback

### PR DESCRIPTION
**Pull request by designertyler**
This incorporates some of the feedback I've heard since the global header first GA'd. Includes small tweaks to overview and definitions

https://carbon-website-git-fork-designertyler-global-header-revi-0908a6.carbon-design-system.now.sh/patterns/global-header

### CODE REVIEW

File: src/pages/patterns/global-header/index.mdx [hunk 1]
```diff
@@ -17,38 +17,41 @@ Users rely on the global header to navigate and orient themselves in your UI. Th
   <AnchorLink>Accessibility</AnchorLink>
   <AnchorLink>Related components</AnchorLink>
   <AnchorLink>References</AnchorLink>
+
 </AnchorLinks>
 
 ## Overview
 
-The global header is essential to a product’s UI. Similar to the menu bar in macOS, it represents the structure of a system and allows access to areas and system functions. This pattern covers the foundation of using UI Shell components for product navigation, and introduces techniques for achieving consistency in products.
+The global header is essential to a product’s UI. It is a consistently available user interface element that contains functionality for the current product as well as for the entire system. Like the macOS Menu Bar and the Windows Start Bar, the global header provides consistent locations to invoke your product’s local navigation as well as pervasive access to system wide functions like System Settings, Notifications, and navigating between products
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
The global header is essential to a product’s UI. It is a consistently available user interface element that contains functionality for the current product as well as for the entire system. Like the macOS Menu Bar and the Windows Start Bar, the global header provides consistent locations to invoke your product’s local navigation as well as pervasive access to system-wide functions like system settings, notifications, and navigating between products.
```

File: src/pages/patterns/global-header/index.mdx [hunk 2]
```diff
@@ -17,38 +17,41 @@ Users rely on the global header to navigate and orient themselves in your UI. Th
...
-| Local       | A local navigation exists within a product's context and provides the means to accomplish product level processes. Local elements will therefore differ from one product to another.                                             |
+| Persistence | Definition                                                                                                                                                                                                                                    |
+| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Global      | A global UI element is preset everywhere in your UI. It contains system and product items your user may need at any time: navigation, authentication, notifications, and more. Global elements remain consistent from one context to another. |
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
| Global      | A global UI element is present everywhere in your UI. It contains system and product items your user may need at any time: navigation, authentication, notifications, and more. Global elements remain consistent from one context to another. |
```

File: src/pages/patterns/global-header/index.mdx [hunk 3]
```diff
@@ -17,38 +17,41 @@ Users rely on the global header to navigate and orient themselves in your UI. Th
...
-| System  | System level processes include navigating to the main sections of a platform and accessing system utilities like notifications or settings. At this level a user can manage the attributes that apply to the whole system. |
-| Product | Product level processes include navigating within a product and accessing the product's core functions. At this level the user directly interacts with the product's main function.                                        |
+| Task    | Definition                                                                                                                                                                                                             |
+| ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| System  | System level tasks include navigating to the main sections of a platform and accessing system utilities like notifications or settings. At this level a user can manage the attributes that apply to the whole system. |
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
| System  | System-level tasks include navigating to the main sections of a platform and accessing system utilities like notifications or settings. At this level a user can manage the attributes that apply to the whole system. |
```

File: src/pages/patterns/global-header/index.mdx [hunk 4]
```diff
@@ -17,38 +17,41 @@ Users rely on the global header to navigate and orient themselves in your UI. Th
...
+| Product | Product level tasks include navigating within a product and accessing the product's core functions. At this level the user directly interacts with the product's main function.                                        |
```
Comment (janchild) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
| Product | Product-level tasks include navigating within a product and accessing the product's core functions. At this level the user directly interacts with the product's main function.                                        |
```

File: src/pages/patterns/global-header/index.mdx [hunk 5]
```diff
@@ -17,38 +17,41 @@ Users rely on the global header to navigate and orient themselves in your UI. Th
   <AnchorLink>Accessibility</AnchorLink>
   <AnchorLink>Related components</AnchorLink>
   <AnchorLink>References</AnchorLink>
+
```
Comment (connor-leech) [inline src/pages/patterns/global-header/index.mdx]: We decided to keep the "Feedback" anchor link for all production patterns. I'd add it in here as a suggestion but I don't think I'll get the indentation right.

File: src/pages/patterns/global-header/index.mdx [hunk 6]
```diff
@@ -144,7 +148,7 @@ This sense of place applies to location as well as states. The header can be use
 
 ### Drill down levels and breadcrumbs
 
-Breadcrumbs let your user see where they are in the hierarchy of the application and gives users a way to navigate back up.
+[Breadcrumbs](https://www.carbondesignsystem.com/components/breadcrumb/code) let your user see where they are in the hierarchy of the application and gives users a way to navigate back up.
```
Comment (connor-leech) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
[Breadcrumbs](/components/breadcrumb/code) let your user see where they are in the hierarchy of the application and gives users a way to navigate back up.
```

This should be a relative link

File: src/pages/patterns/global-header/index.mdx [hunk 7]
```diff
@@ -284,7 +288,7 @@ Navigating between landmark regions helps users who cannot see the visual groupi
 
 </Row>
 
-## Related components
+## Related
 
 - [UI shell header](https://www.carbondesignsystem.com/components/UI-shell-header/code/)
```
Comment (connor-leech) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
- [UI shell header](/components/UI-shell-header/code/)
```

File: src/pages/patterns/global-header/index.mdx [hunk 8]
```diff
@@ -284,7 +288,7 @@ Navigating between landmark regions helps users who cannot see the visual groupi
...
 - [UI shell left panel](https://www.carbondesignsystem.com/components/UI-shell-left-panel/code/)
```
Comment (connor-leech) [inline src/pages/patterns/global-header/index.mdx]:
```suggestion
- [UI shell left panel](/components/UI-shell-left-panel/code/)
```

**Pull request merged**